apps/web/src/components/WalletConnection/ConnectWalletButton.tsx

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable */
```

Comment:
oh, Why we have this?

Reply:
@aleksaToljic I got an error _Promise-returning function provided to attribute where a void return was expected_  for calling connect async function in button onClick event. I didn't find the right solution yet